tests/behat/features/ra_candidate3.feature

```diff
@@ -0,0 +1,43 @@
+Feature: A RAA manages ra candidates from virtual institutions in the ra environment
+  In order to promote candidates from virtual institutions
+  As a RAA
+  I must be able to promote and demote identities were I'm allowed to through the authorization config
+
+  Scenario: Provision an institution and a user to promote and demote later on by an authorized institution
+    Given institution "institution-a.example.com" can "select_raa" from institution "institution-a.example.com"
+      And institution "institution-a.example.com" can "use_ra" from institution "institution-a.example.com"
+      And institution "institution-a.example.com" can "use_raa" from institution "institution-a.example.com"
+
+      And institution "institution-d.example.com" can "select_raa" from institution "institution-a.example.com"
+      And institution "institution-d.example.com" can "select_raa" from institution "institution-b.example.com"
```

Comment:
I do not think adding this explicit institution-b reference adds to the test

Reply:
Good catch, removed it